app/Http/Controllers/Admin/LandingPageController.php

```diff
@@ -28,21 +28,30 @@ class LandingPageController extends ModuleController
 
     protected function indexData($request)
     {
-        return [];
+        $types = LandingPageType::all()->pluck('page_type', 'id')->toArray();
+        $typesOptions = $this->getTypesOptions($types);
+
+        return [
+            'types' => $types,
+            'typesOptions' => $typesOptions,
+        ];
     }
 
     protected function formData($request)
     {
         $types = LandingPageType::all()->pluck('page_type', 'id')->toArray();
         $typesOptions = $this->getTypesOptions($types);
 
+        $item = $this->repository->getById(request('landingPage') ?? request('id'));
+        $baseUrl = '//' . config('app.url') . '/landingpages/' . $item->id . '/';
```

Comment:
The `route()` helper is nice for this sort of thing: https://laravel.com/docs/8.x/urls#urls-for-named-routes